aw_server/rest.py

```diff
@@ -177,35 +179,18 @@ def post(self, bucket_id):
         return event.to_json_dict(), 200
 
 
-# VIEWS
-
-@api.route("/0/views/")
-class ViewListResource(Resource):
-    @copy_doc(ServerAPI.get_views)
-    def get(self):
-        return app.api.get_views(), 200
-
-
-@api.route("/0/views/<string:viewname>")
-class ViewResource(Resource):
-    @api.param("start", "Start datetime of events to query over")
-    @api.param("end", "End datetime of events to query over")
-    @copy_doc(ServerAPI.query_view)
-    def get(self, viewname):
-        args = request.args
-        start = iso8601.parse_date(args["start"]) if "start" in args else None
-        end = iso8601.parse_date(args["end"]) if "end" in args else None
+# QUERY
 
-        result = app.api.query_view(viewname, start, end)
+# It's not possible to expect a raw string in restplus/swagger, stupid
```

Comment:
I think the way to do this is to simply create a query model that contains only one field. Something like this:

```py
query_fields = api.model('Query', {
    'query': fields.String,
})
```

Is there any reason to not do it like this?